src/framework/theme/components/chat/chat.module.ts

```diff
@@ -39,9 +42,11 @@ const NB_CHAT_COMPONENTS = [
   ],
   declarations: [
     ...NB_CHAT_COMPONENTS,
+    NbChatCustomMessageDirective,
   ],
   exports: [
     ...NB_CHAT_COMPONENTS,
+    NbChatCustomMessageDirective,
```

Comment:
Add directive to `NB_CHAT_COMPONENTS` or if you don't want to add it to components array, you could introduce new `NB_CHAT_DIRECTIVES` variable.